Mathlib/RingTheory/Algebraic/Integral.lean

```diff
@@ -636,3 +636,19 @@ open Cardinal in
 end Algebra.IsAlgebraic
 
 end Polynomial
+
+section FractionRing
+
+open Algebra Module
+open scoped nonZeroDivisors
+
+attribute [local instance] FractionRing.liftAlgebra
+
+instance [IsDomain R] [IsDomain S] [NoZeroSMulDivisors R S] [Module.Finite R S] :
```

Comment:
We already have [instFiniteDimensionalFractionRingOfFinite](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/DedekindDomain/IntegralClosure.html#instFiniteDimensionalFractionRingOfFinite) in the general case (it should be moved to a new file and this is my fault, I am not sure why I put it there). Is it worth to add this one to have to import less stuff?